modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -137,9 +144,21 @@ private function decorateSeries($node)
    * sortSeries
    *
    */
-  private function sortSeries(&$series, $sort_order)
+
+
+  private function sortSeries($series, $sort_order, $number)
   {
-    usort($series, function ($a, $b) use ($sort_order) {
+    $no_episode = array_filter($series, function($node) {
+      return $node["episode_id"] === 0;
+    });
+
+    $has_episode = array_filter($series, function($node) {
+      return $node["episode_id"] !== 0;
+    });
+
+
+
+    usort($has_episode, function ($a, $b) use ($sort_order) {
```

Comment:
I'm wondering if a `switch` would work better here? i.e
```
switch($sort_order) {
  case: 'ASC'
    return ...
  case: 'DSC'
    return ...
  default:
    return 0;
}
```
it would maybe make it easier to extend and make the sort order more explicit when reading the code
}
```